docs/settings/reporting-settings.asciidoc

```diff
@@ -206,7 +205,7 @@ When `xpack.reporting.capture.browser.type` is set to `chromium` (default) you c
 [cols="2*<"]
 |===
 | [[xpack-reporting-csv]] `xpack.reporting.csv.maxSizeBytes`
-  | The maximum size of a CSV file before being truncated. This setting exists to prevent
+  | (byte size value) The maximum size of a CSV file before being truncated. This setting exists to prevent
```

Comment:
Same as https://github.com/elastic/kibana/pull/74202/files#r474158774 we should link `byte size value` to https://www.elastic.co/guide/en/elasticsearch/reference/current/common-options.html#byte-units